### PR TITLE
Add auto shim file, allowing clean 'import'

### PR DESCRIPTION
**Pull request by nolde**
This allows the following command to work:

```
import 'promise.prototype.finally/auto'
```

### CODE REVIEW

File: auto.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+var shim = require('./shim');
+module.exports = shim();
```
Comment (ljharb) [inline auto.js]: there's not really a need to export anything from this file; it'd be better if it just does `require('./shim')()`

**Comment by nolde** [comment]
Commit amended to remove export.

**Comment by nolde** [comment]
PR to add this as part of the standard interface for es-shims: https://github.com/es-shims/es-shim-api/pull/16

**Comment by ljharb** [comment]
Thank you!

**Comment by nolde** [comment]
Sorry to disturb you, but do you have any idea about when a new version will be published in NPM carrying this change?

**Comment by ljharb** [comment]
Released as v3.1.0.

**Comment by nolde** [comment]
Sweet!
Thanks a lot!

**Pull request merged**